cms-pages.md

```diff
@@ -198,22 +198,22 @@ They can also be accessed in the markup using the [`this.page` variable](../mark
 
     <p>The title of this page is: {{ this.page.title }}</p>
 
-More information can be found on [`this.page` in the Markup guide](../markup/this-page).
+More information can be found at [`this.page` in the Markup guide](../markup/this-page).
 
 <a name="injecting-assets"></a>
 ## Injecting page assets programmatically
 
-If needed, you can inject assets (CSS and JavaScript files) to pages with the controller's `addCss` and `addJs` methods. It could be done in the `onStart` function defined in the [PHP section](themes#php-section) of a page or [layout](layout) template. Example:
+If needed, you can inject assets (CSS and JavaScript files) into pages with the controller's `addCss` and `addJs` methods. It could be done in the `onStart` function defined in the [PHP section](themes#php-section) of a page or [layout](layout) template. Example:
 
     function onStart()
     {
         $this->addCss('assets/css/hello.css');
         $this->addJs('assets/js/app.js');
     }
 
-If the path specified in the `addCss` and `addJs` method argument begins with a slash (/) then it will be relative to the website root. If the asset path does not begin with a slash then it is relative to the theme.
+If the path specified in the `addCss` and `addJs` method argument begins with a slash (/), it will be relative to the website root. If the asset path does not begin with a slash, it is relative to the theme.
```

Comment:
@DanHarrin IMO, I think this sentence already reads fine as is.

Reply:
@bennothommo I have to stand by this edit, imagine reading the sentence out loud, you would naturally add the commas to improve flow.

Reply:
@DanHarrin yeah, that's fair. :)